### PR TITLE
wip(i18n) add config options and ensure UI supports RTL language direction

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
First pass at ensuring the UI works with both left-to-right and right-to-left languages. (See: https://github.com/frctl/mandelbrot/issues/3)
- Adds two new theme config options:  
  `plugins.web.options.lang` (defaults to `en`)  
  `plugins.web.options.langdir` (defaults to `ltr`)
- Adds four Sass mixins that allow the use of logical values in place of physical ones for `margin-left|right`, `padding-left|right`, `border-left|right` and `left|right`. Basically left is now `start`and right is now `end`.
- Fixes a few tiny bugs, but nothing significant; will leave further changes to subsequent PRs.

_However…_

This PR requires some JavaScript help from @allmarkedup; when text direction is set to `rtl`, the panel resizing behaviour goes a bit crazy, and the navigation sidebar doesn’t collapse/work. Is it easy to test for text-direction in the JS, and adjust behaviour as necessary?

### CODE REVIEW

File: assets/scss/components/_frame.scss [hunk 1]
```diff
@@ -17,21 +17,21 @@
...
-    flex-shrink: 0;
-    height: 50px;
-}
+// .Frame-header {
+//     float: left;
+//     width: 100%;
+//     flex-grow: 0;
+//     flex-shrink: 0;
+//     height: 50px;
+// }
```
Comment (paulrobertlloyd) [inline assets/scss/components/_frame.scss:26]: @allmarkedup I commented out this portion of CSS, and nothing seemed to break! Is it still required?

File: assets/scss/components/_header.scss [hunk 2]
```diff
@@ -9,7 +9,6 @@
     text-overflow: ellipsis;
 }
 .Header-title {
-    padding-left: 15px;
```
Comment (paulrobertlloyd) [inline assets/scss/components/_header.scss:12]: Removed as overridden with later value.

File: assets/scss/components/_header.scss [hunk 3]
```diff
@@ -20,11 +19,11 @@
     // text-overflow: ellipsis;
 }
 .Header-button {
+    @include border-inline('end', 1px solid $color-alpha-tint-light);
     touch-action: manipulation;
     border: 0;
     height: 50px;
     width: 50px;
-    display: none;
```
Comment (paulrobertlloyd) [inline assets/scss/components/_header.scss:27]: Removed as overridden with later value.

File: assets/scss/components/_pen.scss [hunk 4]
```diff
@@ -21,12 +21,12 @@
...
-        padding-left: 10px;
-        margin-left: auto;
+        @include padding-inline('start', 10px);
+        @include margin-inline('start', auto);
     }
     @include mq($until: navCollapse) {
-        padding: 10px 0;
-        margin: 0 15px;
+        padding: 10px 15px;
+        margin: 0;
```
Comment (paulrobertlloyd) [inline assets/scss/components/_pen.scss:29]: Took the opportunity to fix the issue here that we spotted for pen titles on narrower screens.

**Comment by allmarkedup** [comment]
@paulrobertlloyd this is really great mate, many thanks. I'll try look at the JS changes needed later today and then we can get it tested and merged in.

**Pull request merged**